src/app/Fake.Rsync/Rsync.fs

```diff
@@ -0,0 +1,415 @@
+namespace Fake.Rsync
+
+open Fake.Core
+open Fake.IO
+open System
+open System.IO
+
+/// Helpers for running rsync tool
+///
+/// Under windows you will need to add it yourself to your system or use something like cygwin/babun
+///
+/// ## Sample
+///
+/// let result =
```

Comment:
I believe it needs to be indented further in order to be formatted as code in the docs.